docs/en/latest/tutorials/proxy-grpc-service.md

```diff
@@ -72,10 +72,9 @@ kubectl run yages -n ingress-apisix --image smirl/yages:0.1.3 --expose --port 90
 
 Use the service that includes `grpcurl` to test gRPC connectivity.
 
-```bash
-kubectl run -it -n ingress-apisix --rm grpcurl --restart=Never --image=quay.io/mhausenblas/gump:0.1 -- sh
-If you don't see a command prompt, try pressing enter.
-/go $ grpcurl --plaintext yages:9000 yages.Echo.Ping
+```shell
+$ kubectl run -it -n ingress-apisix --rm grpcurl --restart=Never --image=alinsran/grpcurl -- \
```

Comment:
What's the difference between the image?

Reply:
First of all, the original image is no longer maintained and has not been updated for more than five years. When using tls communication, errors will occur. It may be that the protocol content has changed.
I will replace it with the official version 1.8.7, make it work properly.

Reply:
make sense. https://github.com/fullstorydev/grpcurl is good.